src/App.vue

```diff
@@ -76,7 +76,9 @@ export default {
     <main>
       <router-view />
     </main>
+    <footer> <a href="github.com/lukso-network/example-dapp-lsps">github.com/lukso-network/example-dapp-lsps</a></footer>
```

Comment:
The link should start with `https://`

Reply:
sorry about that- also added some href props

https://github.com/lukso-network/example-dapp-lsps/pull/99